card/mod/core/set/all/name.rb

```diff
@@ -97,15 +97,6 @@ def raw_name
   @raw_name || name
 end
 
-def relative_name context_name=nil
-  context_name ||= @supercard.name if @supercard
-  name.name_from context_name
-end
-
-def absolute_name context_name=nil
-  context_name ||= @supercard.name if @supercard
-  name.absolute_name context_name
-end
```

Comment:
REMOVING API (from Card).  I checked WikiRate to try to find any instances where it is used but could not find it.

I got into this area a while back because I was trying to clean up the contextualization stuff for view caching. Most of the name context stuff was discussed and merged in long ago, but I couldn't get rid of these calls until I upgraded the formname handling. 

(by the way, I expect there will be some name context stuff that will be found and fixed this next week, but the work table is much cleaner now)